app/Domains/Employees/Actions/UpdateEmployee.php

```diff
@@ -33,7 +33,14 @@ public function handle(Employee $employee, array $params): Employee
     public function rules(): array
     {
         return [
-
+            'id' => ['required', 'exists:employees,id'],
+            'first_name' => ['sometimes', 'string', 'max:255'],
+            'last_name' => ['sometimes', 'string', 'max:255'],
+            'email' => ['sometimes', 'email', 'unique:employees,email,' . request()->id],
```

Comment:
I'm sorry, I did not understand what you were trying to do here.
Could you please give me a bit more information?

Reply:
Sir I thought the email field will be checked every time a update is made. So I used request()->id to exclude the current email and check with the other emails in the table.